src/PaymentMethodIntegrations/Stripe.php

```diff
@@ -77,7 +77,7 @@ public function register_assets() {
 	 */
 	public function enqueue_data() {
 		$available_gateways = WC()->payment_gateways->get_available_payment_gateways();
-		$stripe_gateway     = $available_gateways['stripe'] ? $available_gateways['stripe'] : null;
+		$stripe_gateway     = isset( $available_gateways['stripe'] ) ? $available_gateways['stripe'] : null;
```

Comment:
Since `::get_available_gateways()` returns a filtered value, I think we should also be doing an `instanceof` check here to make sure it's the Stripe gateway we expect here.

Reply:
`is_a`, because `instanceof` would fail if the class didn't exist.